code/modules/food&drinks/kitchen machinery/deep_fryer.dm

```diff
@@ -71,7 +70,6 @@
 			F.icon = HI
 		if(!(findtext(F.name, "deep fried"))) // deep fried deep fried deep fried deep fried deep fired John Snow says, "hi", no
```

Comment:
Removethis check and the statement after this,just setting the name has no effect and changing real name fucks a lot of things up,so there's no need to keep it, after this the PR is good to go